### DISCUSSION TITLE
how to configure starboard multi-cluster switch?

### DISCUSSION THREAD

**Discussion by fengshunli** [aquasecurity/starboard#742] [Questions & Help]
as the picture shows
![image](https://user-images.githubusercontent.com/18587688/136650620-2226d2b0-73eb-404a-b23b-7f9e6afa4913.png)

**Comment by danielpacak** [comment]
What do you mean by **starboard multi-cluster switch** @fengshunli ? I'm not aware of any such functionality being implemented in Starboard codebase.
